Helper/Setup.php

```diff
@@ -3,12 +3,14 @@
 
 use Magento\Framework\App\Config\ScopeConfigInterface;
 use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\ProductMetadataInterface;
 use Magento\Framework\DB\Ddl\Table;
 use Magento\Framework\Registry;
 use Magento\Framework\Setup\ModuleContextInterface;
 use Magento\Framework\Setup\ModuleDataSetupInterface;
 use Magento\Framework\Setup\SchemaSetupInterface;
 use Magento\Framework\UrlInterface;
+use Magento\Framework\Module\ModuleListInterface;
```

Comment:
Not alphabetical. 😢

Reply:
Every freaking time I can't use basic alphabetical order...

Reply:
@davehensleyns8 Updated in https://github.com/ns8inc/protect-integration-magento/pull/28/commits/48a79b04325f5d7ac74f8dedae99b5e44e27b5ac